spec/plugins/partner_sku_reviews_spec.coffee

```diff
@@ -362,14 +365,27 @@ describe 'PartnerSkuReviews', ->
         beforeEach (done) ->
           prepare 'inline', 'extended', 'no_ratings_id', {}, done
 
-        it 'does not add the plugin style to the head', ->
-          expect(parent_doc.getElementById('sa-partner-sku-reviews-style')).to.not.exist
+        it 'adds the plugin style to the head', ->
+          expect(parent_doc.getElementById('sa-partner-sku-reviews-style')).to.exist
 
         it 'does not render the inline widget', ->
           expect(parent_doc.getElementById('@@flavor-product-reviews-inline').innerHTML.trim()).to.be.empty
 
-        it 'does not render the extended widget', ->
-          expect(parent_doc.getElementById('@@flavor-product-reviews-extended').innerHTML.trim()).to.be.empty
+        it 'renders the extended widget', ->
+          expect(parent_doc.getElementById('@@flavor-product-reviews-extended').innerHTML.trim()).to.not.be.empty
+
+        context 'and prompt is disabled', ->
+          beforeEach (done) ->
+            prepare 'inline', 'extended', 'no_ratings_id', { extended_widget_prompt_enabled: false }, done
+
+          it 'does not add the plugin style to the head', ->
+            expect(parent_doc.getElementById('sa-partner-sku-reviews-style')).to.not.exist
+
+          it 'does not render the inline widget', ->
```

Comment:
[Minor] I think this test is redundant

```
PartnerSkuReviews
  .constructor
    when a reviewable product_id is detected
      and it doesn't have ratings
        ✔ adds the plugin style to the head
        ✔ does not render the inline widget <-----
        ✔ renders the extended widget
        and prompt is disabled
          ✔ does not add the plugin style to the head
          ✔ does not render the inline widget <====
          ✔ does not render the extended widget
```

Reply:
Done, also changed 
`and prompt is disabled` to `and extended widget prompt is disabled`